datalad/core/distributed/clone.py

```diff
@@ -585,15 +585,125 @@ def postclonecfg_ria(ds, props):
+
+            # config_content still None should lead to skip everything special
+            # remote related. Instead enable origin to deliver annex keys via
+            # HTTP:
+            lgr.debug("Trying to enable remote origin for git-annex")
+            ds.repo.enable_remote('origin')
+            ds.repo.fsck(remote='origin', fast=True)
+
+        elif scheme == 'ssh':
+
+            # TODO: switch the following to proper command abstraction:
+            from datalad.distributed.ora_remote import SSHRemoteIO, RIARemoteError
+            # SSHRemoteIO ignores the path part ATM. No remote CWD! (To be
+            # changed with command abstractions). So we need to get that part to
+            # have a valid path to origin's config file:
+            from datalad.support.network import URL
+            from datalad.utils import PurePosixPath
+            cfg_path = PurePosixPath(URL(props['giturl']).path) / 'config'
+            op = SSHRemoteIO(props['giturl'])
+            try:
+                # TODO: double-check path
+                response = op.read_file(cfg_path)
+                config_content = response
+            except RIARemoteError as e:
+                lgr.debug("Failed to get config file from source:\n%s",
+                          exc_str(e))
+
+        elif scheme == 'file':
+
+            # TODO: switch the following to proper command abstraction:
+            from datalad.distributed.ora_remote import LocalIO, RIARemoteError
+            op = LocalIO()
+            from datalad.support.network import URL
+            cfg_path = Path(URL(props['giturl']).localpath) / 'config'
+            try:
+                # TODO: double-check path
+                response = op.read_file(cfg_path)
+                config_content = response
+            except RIARemoteError as e:
+                lgr.debug("Failed to get config file from source:\n%s",
+                          exc_str(e))
+        else:
+            lgr.debug("Unknown URL-Scheme in %s. Can handle HTTP, SSH or FILE."
+                      "", props['source'])
+
+        # 3. And read it
+        org_uuid = None
+        if config_content:
+            from datalad.utils import make_tempfile
+            from datalad.cmd import (
+                GitRunner,
+                WitlessRunner,
+                StdOutCapture
+            )
+
+            with make_tempfile(content=config_content) as cfg_file:
+                runner = WitlessRunner(env=GitRunner.get_git_environ_adjusted())
+                try:
+                    result = runner.run(
+                        ['git', 'config', '-f', cfg_file,
+                         'datalad.ora-remote.uuid'],
+                        protocol=StdOutCapture
+                    )
+                    org_uuid = result['stdout'].strip()
+                except CommandError:
+                    # doesn't contain what we are looking for
+                    lgr.debug("Found nothing in source config.")
```

Comment:
Maybe `Found no UUID for ORA special remote at 'origin'`?

Reply:
Yes, better.